src/Merger/merger_test.go

```diff
@@ -24,22 +25,70 @@ Visual Studio Code,Trial,Infrastructure,false,2,An IDE
 Dagger IO,Assess,Infrastructure,true,1,Its a workflow thing
 Python,Hold,Language,false,0,Its a programming Language`
 
+// The strings below are used exclusively to test
+// ReadCsvFile's majority-vote-based tool-duplication deletion.
+
+var csvfile3 string = `name,ring,quadrant,isNew,moved,description
+Go,Trial,Language,true,0,Its a programming Language
+Visual Studio Code,Trial,Infrastructure,false,2,An IDE
+Dagger IO,Assess,Infrastructure,true,1,Its a workflow thing`
+
+// TODO: Discuss handling ties.
+//var csvfile4 string = `name,ring,quadrant,isNew,moved,description
+//Python,Adopt,Language,false,0,Its a programming Language
+//Visual Studio,Trial,Infrastructure,false,1,An IDE
+//Dagger IO,Assess,Infrastructure,true,1,Its a workflow thing`
+
+var correctMergeMajority string = `name,ring,quadrant,isNew,moved,description
+Go,Adopt,Language,true,0,Its a programming Language
+Visual Studio Code,Trial,Infrastructure,false,2,An IDE
+Dagger IO,Assess,Infrastructure,true,1,Its a workflow thing
+Python,Hold,Language,false,0,Its a programming Language`
+
 var TestFiles []string = []string{"testFile1.csv", "testFile2.csv"}
+var AllTestFiles []string = []string{"testFile1.csv", "testFile2.csv", "testFile3.csv"}
+
+func createSomeCsv(count int) {
+
+	if count > len(AllTestFiles) {
+		log.Fatal("The count given is greater than the number of test files.")
+	}
+	var csvfile = csvfile1
+
+	for i := 0; i < count; i++ {
+		if i == 1 {
+			csvfile = csvfile2
+		} else if i == 2 {
+			csvfile = csvfile3
+		}
+
+		err := os.WriteFile(AllTestFiles[i], []byte(csvfile), 0644)
+		if err != nil {
+			log.Fatal(err)
+		}
+	}
+}
 
 func createCsvFiles() {
-	err := os.WriteFile("testFile1.csv", []byte(csvfile1), 0644)
-	if err != nil {
-		log.Fatal(err)
+	createSomeCsv(2)
+}
```

Comment:
This might as well just be removed an then call `createSomeCsv(2)` where `createCsvFile()` is used